servers/zms/pom.xml

```diff
@@ -196,6 +196,12 @@
       <groupId>software.amazon.awssdk</groupId>
       <artifactId>dynamodb-enhanced</artifactId>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <version>1.4.7</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
why this dependency is needed?
additionally, if it is needed - the version should not be here